nodejs/week2/mealSharing/package.json

```diff
@@ -0,0 +1,19 @@
+{
+	"name": "week1homework",
+	"version": "1.0.0",
+	"description": "",
+	"main": "index.js",
+	"scripts": {
+		"start": "node src/backend/index.js",
+		"dev": "nodemon src/backend/index.js",
+		"test": "echo \"Error: no test specified\" && exit 1"
+	},
+	"author": "",
+	"license": "ISC",
+	"devDependencies": {
+		"nodemon": "^2.0.4"
+	},
+	"dependencies": {
+		"express": "^4.17.1"
+	}
+}
```

Comment:
Interesting! Are you using TABs for indentation instead of spaces? What operative system and editor are you using?